packages/adp-tooling/src/types.ts

```diff
@@ -52,6 +58,23 @@ export interface AdpWriterConfig {
     };
 }
 
+export interface FlpConfiguration {
```

Comment:
This type needs more explanation, please add a short comment to each property so that it is easier to understand what this config is for.

Reply:
Done